libnl-route-3.sym

```diff
@@ -286,6 +286,7 @@ global:
 	rtnl_link_get_broadcast;
 	rtnl_link_get_by_name;
 	rtnl_link_get_carrier;
+	rtnl_link_get_carrier_changes;
```

Comment:
Ah yes, didn't realize that. Will change accordingly (also the other .sym entries)